src/test/java/jewellery/inventory/helper/SaleTestHelper.java

```diff
@@ -147,6 +194,23 @@ private static void calculateProductsDiscounts(Sale sale, SaleResponseDto dto) {
             });
   }
 
+  private static void calculateOrganizationProductsDiscounts(
+      Sale sale, OrganizationSaleResponseDto dto) {
+    sale.getProducts()
+        .forEach(
+            product -> {
+              BigDecimal salePrice =
+                  Optional.ofNullable(product.getSalePrice()).orElse(BigDecimal.ZERO);
+              BigDecimal discount =
+                  Optional.ofNullable(product.getDiscount()).orElse(BigDecimal.ZERO);
+              dto.setTotalDiscountedPrice(dto.getTotalDiscountedPrice().add(salePrice));
+              dto.setTotalDiscount(
+                  (dto.getTotalDiscount().add(discount))
+                      .divide(salePrice, MathContext.DECIMAL128)
+                      .multiply(getBigDecimal("100")));
+            });
```

Comment:
This looks pretty similar to the inner part of `calculateOrganizationResourcesDiscounts`. Can we plug in the salePrice and discount as parameters and extract the highlighted logic into a function?